fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/ContentLocationMessageBodyReader.java

```diff
@@ -61,7 +53,8 @@ public boolean isReadable(final Class<?> type,
                               final Annotation[] annotations,
```

Comment:
This is an array b/c that's part of the javax.ws.rs.ext.MessageBodyReader api: http://docs.oracle.com/javaee/7/api/javax/ws/rs/ext/MessageBodyReader.html